project.godot

```diff
@@ -187,10 +200,10 @@ sokoban_skip={
 2d_physics/layer_8="enemies hitbox"
 2d_physics/layer_9="projectiles"
 2d_physics/layer_10="non_walkable_floor"
-2d_physics/layer_13="hookable"
 
 [rendering]
 
 textures/canvas_textures/default_texture_filter=0
 renderer/rendering_method="gl_compatibility"
 renderer/rendering_method.mobile="gl_compatibility"
+2d/snap/snap_2d_transforms_to_pixel=true
```

Comment:
We deliberately removed this in commit 649af730a7104d19581146a839a0a8d5f6cf7d1b.